backend/coreapp/decompiler_wrapper.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+from coreapp.compiler_wrapper import CompilerWrapper
+from coreapp.m2c_wrapper import M2CError, M2CWrapper
+
+logger = logging.getLogger(__name__)
+
+
```

Comment:
```suggestion
```

Reply:
Classes should have two empty lines before them, per pep8

Reply:
<small>pep8 is gross</small>

We should introduce a code formatter like I have with the TypeScript code like `black` I think